Should structopt re-export `paw`?

`clap-verbosity-flag` is a crate that provides a reusable set of arguments.  When someone tried to use `clap-verbosity-flag` with `paw`, it broke:
```
error[E0433]: failed to resolve: use of undeclared type or module `paw`
  --> /Users/edpage/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-verbosity-flag-0.3.1/src/lib.rs:21:10
   |
21 | #[derive(structopt::StructOpt, Debug, Clone)]
   |          ^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `paw`

error: aborting due to previous error
```

`structopt` has the `paw` feature on but the symbol `paw` isn't visible within `clap-verbosity-flag` to be able to compile.

Currently structopt re-exports clap.  Should it also re-export paw?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,7 +21,7 @@ lints = ["clap/lints"]
 debug = ["clap/debug"]
 no_cargo = ["clap/no_cargo"]
 doc = ["clap/doc"]
-paw = ["structopt-derive/paw"]
+paw = ["structopt-derive/paw", "paw_dep"]
 
 [badges]
 travis-ci = { repository = "TeXitoi/structopt" }
@@ -30,6 +30,7 @@ travis-ci = { repository = "TeXitoi/structopt" }
 clap = { version = "2.33", default-features = false }
 structopt-derive = { path = "structopt-derive", version = "=0.4.8" }
 lazy_static = "1.4.0"
+paw_dep = { version = "1", optional = true, package = "paw" }
 
 [dev-dependencies]
 trybuild = { version = "1.0.5", features = ["diff"] }

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1063,6 +1063,8 @@ use std::ffi::OsString;
 
 /// Re-exports
 pub use clap;
+#[cfg(feature = "paw")]
+pub use paw_dep as paw;
 
 /// **This is NOT PUBLIC API**.
 #[doc(hidden)]

--- a/structopt-derive/src/lib.rs
+++ b/structopt-derive/src/lib.rs
@@ -731,7 +731,7 @@ fn gen_from_subcommand(
 #[cfg(feature = "paw")]
 fn gen_paw_impl(name: &Ident) -> TokenStream {
     quote! {
-        impl paw::ParseArgs for #name {
+        impl ::structopt::paw::ParseArgs for #name {
             type Error = std::io::Error;
 
             fn parse_args() -> std::result::Result<Self, Self::Error> {

